tests/hil/scripts/pytest-hil/nrf52840dk.py

```diff
@@ -2,6 +2,12 @@
 from nordicboard import NordicBoard
 
 class nRF52840DK(ZephyrBoard, NordicBoard):
+    def program(self, *args, **kwargs):
```

Comment:
Wrong class?

Reply:
Indeed, sorry about that. Rerunning tests after correcting this.